src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -117,13 +117,20 @@ void EditShortcutModel::checkNewSequenceForConflicts()
 {
     m_conflictShortcut.clear();
     QString input = newSequence();
+    std::vector<std::string> inputSequences = Shortcut::sequencesFromString(input.toStdString());
```

Comment:
This conversion is not needed because we no longer allow the user to assign multiple shortcuts to one action